src/main/resources/swagger/agora.yaml

```diff
@@ -139,7 +140,7 @@ paths:
         - Methods
       responses:
         200:
-          description: An array of methods.
+          description: An single method.
           schema:
             type: array
```

Comment:
Getting a single method does not return an array. Might as well change this since we're here